i6_models/parts/mha.py

```diff
@@ -0,0 +1,80 @@
+__all__ = ["MultiheadAttentionV1", "MultiheadAttentionV1Config"]
+
+import math
+
+import torch
+from torch import nn
+from i6_models.config import ModelConfiguration
+from i6_models.util import compat
+from dataclasses import dataclass
+
+
+@dataclass
+class MultiheadAttentionV1Config(ModelConfiguration):
```

Comment:
Docstring please, can use the one from `ConformerMHSAV1Config`

Reply:
Added.